dev/tests/integration/testsuite/Magento/Wishlist/Model/WishlistTest.php

```diff
@@ -116,17 +116,13 @@ public function testGetItemCollection(): void
     }
 
     /**
-     * @magentoDataFixture Magento/Wishlist/_files/wishlist.php
+     * @magentoDataFixture Magento/Wishlist/_files/wishlist_with_disabled_simple_product.php
      *
      * @return void
      */
     public function testGetItemCollectionWithDisabledProduct(): void
     {
-        $productSku = 'simple';
         $customerId = 1;
-        $product = $this->productRepository->get($productSku);
```

Comment:
@skovalenk  here we have another test:
Originally we tested, that product, that was added to Wishlist will not be available to Customer if it was disabled

Now, you test disabled product added to wishlist will not be available to Customer.

Reply:
I've changed wishlist_with_disabled_simple_product.php fixture in order to keep previous behaviour